autoload/neomake/makers/typescript_project.vim

```diff
@@ -0,0 +1,13 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#typescript_project#typescript_project()
+    return {
+        \ 'exe': 'tsc',
+        \ 'args': ['--noEmit'],
+        \ 'errorformat':
+            \ '%E%f %#(%l\,%c): error %m,' .
+            \ '%E%f %#(%l\,%c): %m,' .
+            \ '%Eerror %m,' .
+            \ '%C%\s%\+%m'
+        \ }
+endfunction
```

Comment:
I see.
Can you update the PR to only include that commit, please?
You basically need to `git rebase -i origin/master`, delete the other commit and then `git push -f` to the branch for this PR.

(I was not aware of project makers - but thought that files below `ft/` would be library functions)